app/client/components/contactUs/contactUs.tsx

```diff
@@ -3,12 +3,13 @@ import { Button } from "@guardian/src-button";
 import { palette, space } from "@guardian/src-foundations";
 import { headline, textSans } from "@guardian/src-foundations/typography";
 import { RouteComponentProps } from "@reach/router";
+import * as Sentry from "@sentry/browser";
```

Comment:
```suggestion
import { captureException } from "@sentry/browser";
```

Reply:
it looks like `captureException` is the only part of Sentry that you are using here, so perhaps we could just import that and call it like this:
`captureException(errorMsg);`

instead of:
`Sentry.captureException(errorMsg);`